client/signup/site-mockup/style.scss

```diff
@@ -66,13 +71,46 @@
 	}
 }
 
+// Some microcopy to help people understand the mockups.
+.site-mockup__help-tip {
+	color: var( --color-primary-50 );
+	text-align: center;
+	font-size: 13px;
+	font-weight: 300;
+	margin: 24px 0 -8px 0;
+	transition: all 300ms cubic-bezier(.08,.74,.44,1.07);
+	transition-delay: 200ms;
+
+	p {
```

Comment:
Looking on mobile, we might need some margins on the sides here:

![image](https://user-images.githubusercontent.com/6981253/54214761-375f2200-44bd-11e9-8c76-36f68933bdd9.png)

Reply:
Addressed in 374aae8.